fs/osfs.py

```diff
@@ -363,7 +366,9 @@ def setinfo(self, path, info):
                     with convert_os_errors('setinfo', path):
                         os.utime(sys_path, (accessed, modified))
 
-    def _scandir(self, path, namespaces=None):
+
+
+    def _scandir_c(self, path, namespaces=None):
```

Comment:
At first that's what I wanted to do, but for the sake of testing it's easier to have the two methods accessible at all times, since your way involves reloading the module while mocking an import to be able to access the other implementation. But I'll give it a shot anyway.